src/Graphics/UI.h

```diff
@@ -75,7 +75,7 @@ class UI
     int _y = 0;
     int _xOffset = 0;
     int _yOffset = 0;
-    Texture* _texture = 0;
+    mutable Texture* _texture = 0;
```

Comment:
Because in some inherited classes _texture object is created in texture() method. **Const** specifier don't allow this